python/lsst/pipe/tasks/scaleZeroPoint.py

```diff
@@ -43,6 +46,11 @@ class ImageScaler:
     def __init__(self, scale=1.0):
         self._scale = scale
 
+    @property
+    def scale(self) -> float:
+        """Scale that it applies to a specified image."""
+        return self._scale
+
```

Comment:
I'm not sure if there's any utility in keeping an instance's `scale` unmodified once created.

Reply:
When in doubt, less mutability tends to be better.